.github/workflows/ci.yaml

```diff
@@ -61,4 +61,11 @@ jobs:
 
       - name: Run tests
         run: |
-          vendor/bin/phpunit
+          vendor/bin/phpunit --coverage-clover build/logs/clover.xml
+
+      - name: Upload coverage results to Coveralls
+        continue-on-error: true
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          php vendor/bin/php-coveralls --coverage_clover=build/logs/clover.xml -v
```

Comment:
I'm OK to merge this PR as is, but you may want to set [COVERALLS_PARALLEL](https://github.com/php-coveralls/php-coveralls#parallel-builds). Right now, each job is reporting the coverage on it's own, but it also creates different builds on coveralls.io level, eg https://coveralls.io/builds/35154491 and https://coveralls.io/builds/35154494

Reply:
I think this isn't the reason, but: https://github.com/php-coveralls/php-coveralls/pull/305

Reply:
how coveralls will know how many parallel jobs to receive, @sanmai ?
it's sdk client independent, @sanmai 
https://docs.coveralls.io/parallel-build-webhook

Reply:
It figures this out somehow on Travis CI, isn't it?

Reply:
@sanmai , no, it does not figures out somehow automagically, someone has to call the hook, as documented in official docs.